### ISSUE TITLE
PaginatorState not properly initialized when used in Table with lazy loading

### ISSUE THREAD

**Issue by mricciuti** [primefaces/primeng#5191]
**I'm submitting a bug report **  (check one with "x")
```
[x ] bug report => Search github for a similar issue or PR before submitting
[ ] feature request => Please check if request is not on the roadmap already https://github.com/primefaces/primeng/wiki/Roadmap
[ ] support request => Please do not submit support request here, instead see http://forum.primefaces.org/viewforum.php?f=35
```

**Plunkr Case (Bug Reports)**
See this plunkr:    http://plnkr.co/edit/StZoOcX9WKfgdQKRD55q?p=preview

**Current behavior**
There is an issue when using Turbo Table with  both paginator and lazy loading features:
the first time the table is displayed and initialized with data, the totalRecords property is not correctly propagated from the table to the paginator state variable. So if we want to access the "paginatorstate.totalRecords' property to build a left/right paginator detail template, we get an undefined value when loading the first page of the table.

Note: same problem is reproduced when using the old DataTable component, so this issue is not specific to Turbo Table.

**Expected behavior**
When the data are lazy loaded the first time at table initialization, the totalRecord property should be properly propagated to paginatorState variable

**Minimal reproduction of the problem with instructions**
See Plunkr  http://plnkr.co/edit/StZoOcX9WKfgdQKRD55q?p=preview : 
 - run the app, table will be displayed with a 1sec tempo to simulate lazy loading
   => see in paginator right detail:  the expected TotalRecords value is not displayed as expected (undefined)
 - navigator to page 2 with the paginator
   => now the TotalRecords property is correctly displayed in the paginator right detail panel.

**Please tell us about your environment:**
OS: Windows 7 /  IDE: VisualStudioCode

* **Angular version:**  
5.2.0

* **PrimeNG version:** 
5.2.0

* **Browser:** 
All

**Comment by mricciuti** [comment]
We can see that paginator links are correctly created/updated after the initial loading of first page. So it seems that after first "lazy load" event is emitted and data is fetched, the method "paginator. updatePageLinks()" is called as expected , but not "paginator.updatePaginatorState".  
Hope this helps

**Comment by cagataycivici** [comment]
Should be fixed in 5.2.4.

**Comment by stephanie-dm** [comment]
I can still reproduce this issue with 5.2.7. The suggested workaround works though.
